class/UserAccessManager.php

```diff
@@ -953,38 +960,63 @@ public function endsWith($sHaystack, $sNeedle)
      */
     
     /**
-     * The function for the wp_print_styles action.
+     * Register styles and scripts with handle for admin panel.
      */
-    public function addStyles()
-    {
-        wp_enqueue_style(
-            'UserAccessManagerAdmin', 
-            UAM_URLPATH . "css/uamAdmin.css",
+    protected function registerAdminStylesAndScripts() {
+        wp_register_style(
+            self::HANDLE_STYLE_ADMIN,
+            UAM_URLPATH . 'css/uamAdmin.css',
             array() ,
-            '1.0',
+            $this->_sUamVersion,
             'screen'
         );
-        
-        wp_enqueue_style(
-            'UserAccessManagerLoginForm', 
-            UAM_URLPATH . "css/uamLoginForm.css",
-            array() ,
-            '1.0',
-            'screen'
+        wp_register_script(
+            self::HANDLE_SCRIPT_ADMIN,
+            UAM_URLPATH . 'js/functions.js',
+            array('jquery'),
+            $this->_sUamVersion
         );
     }
-    
+
     /**
-     * The function for the wp_print_scripts action.
+     * The function for the admin_enqueue_scripts action for styles and scripts.
+     * @param string $hook 
      */
-    public function addScripts()
+    public function enqueueAdminStylesAndScripts($hook)
     {
-        wp_enqueue_script(
-            'UserAccessManagerFunctions', 
-            UAM_URLPATH . 'js/functions.js', 
-            array('jquery')
+        $this->registerAdminStylesAndScripts();
+        wp_enqueue_style(self::HANDLE_STYLE_ADMIN);
+        if ($hook == 'uam_page_uam_settings') {
+            wp_enqueue_script(self::HANDLE_SCRIPT_ADMIN);
+        }
+    }
+
+    /**
+     * Functions for other content.
+     */
+
+    /**
+     * Register all other styles.
+     */
+    protected function registerStyles() {
+        wp_register_style(
+            self::HANDLE_STYLE_LOGIN_FORM,
+            UAM_URLPATH . 'css/uamLoginForm.css',
+            array(),
+            $this->_sUamVersion,
+            'screen'
         );
     }
+
+    /**
+     * The function for the wp_enqueue_scripts action.
+     */
+    public function enqueueStyles()
```

Comment:
Accordingly to admin function I would name it enqueueStylesAndScripts even if it currently only contains styles.